edgedb-protocol/src/query_arg.rs

```diff
@@ -526,3 +527,99 @@ implement_tuple! {9, T0, T1, T2, T3, T4, T5, T6, T7, T8, }
 implement_tuple! {10, T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, }
 implement_tuple! {11, T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, }
 implement_tuple! {12, T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, }
+
+
+
+pub struct ValueWithCardinality(Option<Value>, Cardinality);
+
+impl<V: Into<Value>> From<V> for ValueWithCardinality {
+	fn from(value: V) -> Self {
+		ValueWithCardinality(Some(value.into()), Cardinality::One)
+	}
```

Comment:
There are lots of hard tabs; I think the codebase uses 4 space indents.
(... shouldn't there be a linter?)

Reply:
> There are lots of hard tabs; I think the codebase uses 4 space indents. (... shouldn't there be a linter?)

Oops. It should be. But seems like it's missing! I will convert to spaces.